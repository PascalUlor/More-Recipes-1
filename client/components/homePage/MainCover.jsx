import React from 'react';
import { Link } from 'react-router-dom';

const MainCover = () => (
  <main className="landing">
    <div className="overlay">
      <div className="container">
        <section className="cover-caption">
          <h1 className="display-3">Share Your Special Recipe Ideas Instantly</h1>
          <p id="sub-title" className="lead">More-Recipes provides a platform for users to share the awesome and exciting recipe ideas they have invented or learnt, get feedback in form of reviews and votes from other users who explore that recipe</p>
          <Link className="btn btn-lg btn-warning mt-3" to="/signup" role="button">Get Started</Link>
          <p className="lead beautify">Already Registered?
            <Link className="btn btn-link" to="/signin" role="button">Sign In</Link>
          </p>
        </section>
      </div>
    </div>
  </main>
);

export default MainCover;