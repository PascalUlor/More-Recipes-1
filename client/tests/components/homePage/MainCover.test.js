/* eslint-disable */
import React from 'react';
import toJson from 'enzyme-to-json';
import MainCover from '../../../components/homePage/MainCover.jsx';


describe('<Footer/>', () => {
  it('renders footer component without crashing', () => {
    const shallowWrapper = shallow(<MainCover/>);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
