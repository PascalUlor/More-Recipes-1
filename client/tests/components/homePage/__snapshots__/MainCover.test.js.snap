// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders footer component without crashing 1`] = `
<main
  className="landing"
>
  <div
    className="container"
  >
    <section
      className="cover-caption"
    >
      <h1
        className="display-3"
      >
        Share Your Special Recipe Ideas Instantly
      </h1>
      <p
        className="lead"
        id="sub-title"
      >
        More-Recipes provides a platform for users to share the awesome and exciting recipe ideas they have invented or learnt, get feedback in form of reviews and votes from other users who explore that recipe
      </p>
      <Link
        className="btn btn-lg btn-warning mt-3"
        replace={false}
        role="button"
        to="/signup"
      >
        Get Started
      </Link>
      <p
        className="lead beautify"
      >
        Already Registered?
        <Link
          className="btn btn-link"
          replace={false}
          role="button"
          to="/signin"
        >
          Sign In
        </Link>
      </p>
    </section>
  </div>
</main>
`;
